tracking/imu/imu.c

```diff
@@ -1,29 +1,85 @@
 #include "imu.h"
-#include <furi_hal.h>
 
-bool bmi160_begin();
-int bmi160_read(double* vec);
+imu_t imu_types[] = {
```

Comment:
Listing all functions here is a bit excessive. I would suggest every header to expose just an `external struct imu_t` and define them in the corresponding .c files, then you wouldn't need to list them all twice (or define in the .h file in the first place), would make slightly better encapsulation IMHO.

Reply:
(TAG names and addresses are probably better to be kept in .c files as well. Note that you can reuse TAG as a device name, too.)